### PR TITLE
Create Risk Premium Adjustment for ETH-A Vault.md

### PR DESCRIPTION
**Pull request by ahbartsch**
@LongForWisdom

### CODE REVIEW

File: governance/polls/Risk Premium Adjustment for ETH-A Vault.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
```
Comment (LongForWisdom) [inline governance/polls/Risk Premium Adjustment for ETH-A Vault.md:1]: Filename should match title: Adjust the ETH-A Risk Premium - September 21, 2020.md

File: governance/polls/Risk Premium Adjustment for ETH-A Vault.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+title: Adjust the ETH-A Risk Premium - September 21, 2020
+summary: Signal your support for increasing the ETH-A Risk Premium to from 0% to 2%
```
Comment (LongForWisdom) [inline governance/polls/Risk Premium Adjustment for ETH-A Vault.md:3]: support or opposition

File: governance/polls/Risk Premium Adjustment for ETH-A Vault.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+---
+# Poll: ETH-A Risk Premium Adjustment - September 21, 2020
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Maker Governance Community. The community can use this poll to express support or opposition to increasing the ETH-A Risk Premium from 0% to 2%.
+
+This Governance Poll ([FAQ](https://community-development.makerdao.com/makerdao-scd-faqs/scd-faqs/governance)) will be active for three days beginning on Monday, September 21 at 16:00 UTC, the results of which may inform an Executive Vote which will go live on Friday, September 25, at 16:00 UTC.
+
+## Review
+
+The ETH-A Risk Premium was voted on in a forum signal request. Please review the [Thread](https://forum.makerdao.com/t/signal-request-increase-eth-a-risk-premium-september-2020/4042) to inform your position before voting.
```
Comment (LongForWisdom) [inline governance/polls/Risk Premium Adjustment for ETH-A Vault.md:19]: super minor. Lowercase t for Thread.

**Review by LongForWisdom — CHANGES_REQUESTED** [review]
Mostly great. See comments for suggested minor changes. Thanks Aaron.

**Comment by ahbartsch** [comment]
Donezo.

**Review by LongForWisdom — APPROVED** [review]
Thanks Aaron!

**Pull request merged**